sonarr/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,22 @@
+name: Sonarr
+files:
+- name: sonarr.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/default
```

Comment:
I suggest moving `template: instances/default` to the bottom of the options so you can have `url` and `api_key` appear on the top, since the template is pretty big.